hw/ip/lc_ctrl/doc/lc_ctrl_function_signals_table.md

```diff
@@ -53,7 +53,7 @@
   </tr>
   <tr>
     <td style="text-align:left;color:red">POST_TRANSITION</td>
-    <td colspan="7" style="background:#dadce0"></td><td>Y*</td><td>Y</td>
+    <td colspan="7" style="background:#dadce0"></td><td>Y*</td><td>Y*</td>
```

Comment:
one quick question:
should this be N? If the original state is Invalid or Esacalate_en, can LC_CTRL still request a state transition?

Reply:
Good catch. I have updated this entire column now. `lc_escalate_en` is not a conditionally asserted signal anymore since we have changed the escalation behavior. Rather, it is only asserted in the invalid / scrap and escalate states.